packages/remix-netlify-edge/package.json

```diff
@@ -16,9 +16,6 @@
   "dependencies": {
     "@remix-run/deno": "1.8.0"
   },
-  "devDependencies": {
-    "@netlify/edge-functions": "^2.0.0"
-  },
```

Comment:
We should keep it in our `devDependencies`, otherwise we won't have it installed for local development

We're doing the same thing in our other adapters btw, so maybe looking at them can help you figure some things out.